corehq/apps/analytics/templates/analytics/initial/appcues.html

```diff
@@ -0,0 +1,22 @@
+{% load hq_shared_tags %}
+
+{% if ANALYTICS_CONFIG.DEBUG %}
```

Comment:
These values don't need to be inside an if statement. The saas check and debug mode are controlled by [global.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/analytics/templates/analytics/initial/global.html) plus the fact that the Appcues id is only available on the prod server. The 31 days check is specific to Drift.

Reply:
Ah, I missed that.

Reply:
Though it'd simplify the code a bit to only put the `apiId` inside the logic and not duplicate the other values.

Reply:
That makes sense. It should be fine to just send the other values in no matter what